### PR TITLE
fix(CMS): Log unsupported data for show_paragraph?/2

### PR DESCRIPTION
**Pull request by kotva006**
#### Summary of changes\
No Ticket.  On Dev Green the landing page was failing to load due to an unsupported type of paragraph.  This will log it, and allow the application to continue to function.

---

#### General checks
* [ ] **Are the changes organized into self-contained commits with descriptive and well-formatted commit messages?** This is a good practice that can facilitate easier reviews.
* [ ] **Testing.** Do the changes include relevant passing updates to tests? This includes updating screenshots. Preferably tests are run locally to verify that there are no test failures created by these changes, before opening a PR.
* [ ] **Tech debt.** Have you checked for tech debt you can address in the area you're working in? This can be a good time to address small issues, or create Asana tickets for larger issues.

#### New UI, or substantial UI changes
* [ ] **Cross-browser compatibility** is less of an issue now that we're no longer supporting IE, but changes still need to work as expected in Safari, Chrome, and Firefox.
* [ ] **Are interactive elements accessible?** This includes at minimum having [relevant keyboard interactions](https://thinkdobecreate.com/articles/4-required-tests-before-shipping-new-features/#keyboard-interaction) and [visible focus](https://thinkdobecreate.com/articles/4-required-tests-before-shipping-new-features/#visible-focus), but can also include verification with screen reader testing.
* [ ] **Other accessibility checks** such as sufficient [color constrast](https://thinkdobecreate.com/articles/4-required-tests-before-shipping-new-features/#color-contrast), or whether the layout holds up at 200% [zoom level](https://thinkdobecreate.com/articles/4-required-tests-before-shipping-new-features/#visible-focus).

#### New endpoints, or non-trivial changes to current endpoints
* [ ] **Have we load-tested** any new pages or internal API endpoints that will receive significant traffic? [See load testing docs](./../load_tests/README.md)
* [ ] **If this change involves routes,** does it work correctly with pertinent "unusual" routes such as the combined Green Line, Silver Line, Foxboro commuter rail, and single-direction bus routes like the 170?

### CODE REVIEW

File: lib/cms/helpers.ex [hunk 1]
```diff
@@ -3,6 +3,7 @@ defmodule CMS.Helpers do
   Various helper functions that aid in parsing CMS JSON data.
   """
 
+  require Logger
```
Comment (anthonyshull) [inline lib/cms/helpers.ex:6]: I like to separate out the different include calls. So:

```
require Logger

alias CMS.API
```

**Review by anthonyshull — APPROVED** [review]

**Review by thecristen — CHANGES_REQUESTED** [review]
The home page is broken.

**Comment by amaisano** [comment]
I was using Dev green to double check that the new Drupal 9 CMS branch was having the same problems as my local.

If you found the issue I'd love to work with you. It's definitely from a CMS repo change.

**Comment by thecristen** [comment]
image

The home page is no longer having a server error, and the bad underlying data is reflected in this way.

**Comment by anthonyshull** [comment]
@thecristen @kotva006 pretty sure this can just be closed since the issue was from CMS changes

**Comment by amaisano** [comment]
I am still hacking away at the issue on the CMS repo however. Definitely an unexpected regression with D9's REST API.

**Comment by amaisano** [comment]
> @amaisano How's that going? Noticing the dev-green is still having these issues.

Haven't made too much progress. Would you like me to deploy the stable master branch back to the sandbox, so dev-green is normal?

**Pull request closed without merging**